docs/DEVELOPING.md

```diff
@@ -0,0 +1,121 @@
+# Development Workflow
+
+Local workflow for running the Adventure Simulator demo.
+
+## Architecture Overview
+
+The game uses a Mount & Blade style architecture:
+
+1. **Strategic Layer** (SpacetimeDB) - Character progression, inventory, mission tracking
+2. **Tactical Spawner** - Watches for pending missions, spawns tactical servers
+3. **Tactical Server** (Lightyear) - Real-time 3D gameplay server
```

Comment:
Both of those should be *SpacetimeDB Clients* which would make our lives easier even more